packages/sdk/src/evm/core/classes/contract-publisher.ts

```diff
@@ -337,11 +337,29 @@ export class ContractPublisher extends RPCConnectionHandler {
     invariant(signer, "A signer is required");
     const publisher = await signer.getAddress();
 
-    const predeployMetadata = await fetchRawPredeployMetadata(
+    const predeployMetadata = await fetchPreDeployMetadata(
```

Comment:
careful with this. I think i remember there was a reason why i needed the raw metadata here instead of the fully fetched one

Reply:
Extracted this change to different branch